client/src/pages/private/ParticipantEOI.js

```diff
@@ -283,18 +278,17 @@ export default () => {
                       {!enableEdit ? 'Edit Info' : 'Done Edit'}
                     </Button>
                   </Grid>
-                  {!disableConfirmInterest && (
-                    <Grid item>
-                      <Button
-                        variant='contained'
-                        onClick={() => {
-                          setOpenConfirmInterest(true);
-                        }}
-                      >
-                        Reconfirm interest
-                      </Button>
-                    </Grid>
-                  )}
+                  <Grid item>
+                    <Button
+                      disabled={disableWithdraw}
```

Comment:
What happen when user already reconfirmed?

Reply:
We should leave this be. From talking with Kendall, the client wants users to have the ability to reconfirm interest as much as they want.